share/eds/testing.go

```diff
@@ -484,3 +493,34 @@ func checkPowerOfTwo(n int) bool {
 	}
 	return n&(n-1) == 0
 }
+
+func TestAccessorSampling(t *testing.T) {
+	ctx := context.Background()
+	acc := NewRandAccessor(t, 8)
+	defer acc.Close()
+
+	size, err := acc.Size(ctx)
+	require.NoError(t, err)
+
+	for squareHalf := 0; squareHalf < 2; squareHalf++ {
+		for axisType := range []rsmt2d.Axis{rsmt2d.Row, rsmt2d.Col} {
+			_, err := acc.AxisHalf(ctx, axisType, size/2*(squareHalf))
+			require.NoError(t, err)
+		}
+	}
+}
```

Comment:
Could you clarify what are you trying to test here? This test seems to exists as part of accessor testing suite in func `testAccessorSample`.